### ISSUE TITLE
Go through groups listed on https://angular.de/meetups/ and add missing ones

### ISSUE THREAD

**Issue by karol-depka** [voidcosmos/angular-communities#72]
Hi Again :).

There is this list:
https://angular.de/meetups/

Most of them are already added, but I've noticed some missing:
https://www.meetup.com/es/ngGirls-RheinNeckar/ (last meetup on 19 enero, counts as active? :), has shield)
https://www.meetup.com/es/Angular-Kiel/ (active, didn't find shield yet)
There could be more missing, I could go through all of them and check.

In general, congrats for having such a vast majority of them already! :)

I can take this task if approved :).

Saludos,
k

**Comment by karol-depka** [comment]
Another one: https://www.meetup.com/es/niederrhein-valley/ but there is another one in Niederrhein and this one does not have Angular in the name, although it has something to do with Angular :) - what is the policy - has to have angular/ng in name? :)

**Comment by Caballerog** [comment]
Of course, if their communities are alive, and we have the information of their organizers, place, etc ... It would be wonderful if the information is included through a PR.

Of course, a different PR for each of the communities so that it can be reviewed individually.

regards!
